packages/map-template/src/hooks/useMapBoundsDeterminer.js

```diff
@@ -70,7 +70,7 @@ const useMapBoundsDeterminer = () => {
      */
     useEffect(() =>  {
         determineMapBounds();
-    }, [mapsIndoorsInstance, currentVenueName, venues, locationId, kioskOriginLocationId, pitch, bearing, startZoomLevel, categories]);
```

Comment:
We don't need the current venue here anymore?

Reply:
No, we only need the venue (the value of the venue prop that is externally exposed).

Reply:
Alright 😄